components/cdc/src/endpoint.rs

```diff
@@ -1056,7 +1066,11 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
                 };
 
             if !regions.is_empty() {
-                match scheduler.schedule(Task::MinTS { regions, min_ts }) {
+                match scheduler.schedule(Task::MinTS {
+                    regions,
+                    min_ts,
+                    current_ts: min_ts_pd,
```

Comment:
what the value of `min_ts_pd` if `get_tso` failed.  if it's 0, should it be current physical time on this node?